web/modules/custom/dpl_fbi/dpl_fbi.install

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use Drupal\Core\Database\Database;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use function Safe\sprintf as sprintf;
+
+/**
+ * Adds 'material_type' column to 'dpl_fbi_work_id' field storage.
+ *
+ * This update function alters the schema for custom field storage
+ * to include a new column for material type. It handles both the
+ * base table and the revision table for the field.
+ *
+ * The function is taken from drupal stackexchange:
+ * https://drupal.stackexchange.com/questions/255495/add-column-to-custom-field-schema
```

Comment:
We are beginning to build a collection of functions that will help us with update functions. As some point it would be nice to collect these in a helper module/package.

See also #843.